tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +50,39 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
```

Comment:
Please add here info log that test checks Azure EH properties

Reply:
Added logs for this step